### PR TITLE
Add reference to window-controls-overlay to the display_override members

### PR DESCRIPTION
**Pull request by sonkkeli**
***
<a href="https://pr-preview.s3.amazonaws.com/sonkkeli/manifest-incubations/pull/60.html" title="Last updated on Sep 8, 2022, 6:36 PM UTC (d072730)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WICG/manifest-incubations/60/b2bf53c...sonkkeli:d072730.html" title="Last updated on Sep 8, 2022, 6:36 PM UTC (d072730)">Diff</a>

### CODE REVIEW

**Comment by marcoscaceres** [comment]
It might just be worth merging this and then for me to do a big cleanup of the referencing.

**Comment by sonkkeli** [comment]
[marcoscaceres](https://github.com/marcoscaceres) that's alright for me :)

File: index.html [hunk 1]
```diff
@@ -49,15 +55,18 @@ <h2 data-dfn-for="">
...
       <p>
         The <dfn>display_override</dfn> member of the [=application manifest=]
         is a <a>sequence</a> of <a data-cite=
-        "appmanifest#dfn-display-modes-values">display mode values</a>. This
-        item represents the developer's preferred fallback chain for
-        <a data-cite="appmanifest#dfn-display-mode">display modes</a>. This
-        field overrides the [=manifest/display=] member. If the user agent does
+        "appmanifest#dfn-display-modes-values">display mode values</a> including
+        extensions like <a data-cite="
+        window-controls-overlay#addition-of-window-controls-overlay-to-the-manifest">
```
Comment (dmurph) [inline index.html]: Can you spend a bit of time trying to see if you can get this to link with [=...=] syntax or [[..]] syntax?

Here is a cheat sheet: https://respec.org/docs/#Shorthands-Guide

Comment (dmurph) [reply index.html]: (here and below)

**Comment by sonkkeli** [comment]
The references should now work after https://github.com/WICG/manifest-incubations/pull/61 and https://github.com/WICG/window-controls-overlay/pull/63 are merged. Will test when they are and ping here then.

File: index.html (hunk 1 above)
Comment (sonkkeli) [reply index.html]: Just lacked a couple of data-exports on the WCO dfns and specrefs for WCO and manifest-incubations

**Comment by dmurph** [comment]
LGTM

**Comment by sonkkeli** [comment]
This is also now rebased and can be merged after https://github.com/WICG/window-controls-overlay/pull/63 has been merged.

**Pull request merged**